static/js/formatters/generate-cta-field-type-link.js

```diff
@@ -10,7 +10,7 @@ export function generateCTAFieldTypeLink(cta) {
   }
   const normalizedCTA = {
     ...cta,
-    linkType: normalizeCtaLinkType(cta.linkType)
+    linkType: normalizeCtaLinkType(cta.linkType || 'URL')
```

Comment:
can we add a comment indicating that 'URL' will use the link with no formatting applied?

Reply:
updated! I put it in the function's main jsdoc instead of the function body